src/experiment_prototype/experiment_utils/decimation_scheme.py

```diff
@@ -126,6 +126,17 @@ def __init__(self, rxrate, output_sample_rate, stages=None):
                 )
                 raise ValueError(errmsg)
 
+            if (
+                self.input_rates[stage_num]
+                > self.dm_rates[stage_num] * self.output_rates[stage_num]
```

Comment:
Right. I should have known that since that was what I was thinking about anyways when I was first trying to figure out this check. For now marking as a TODO will do